packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -35,9 +36,25 @@ export default function LetterSpacingControl( {
 		availableUnits: availableUnits || [ 'px', 'em', 'rem' ],
 		defaultValues: { px: 2, em: 0.2, rem: 0.2 },
 	} );
+
+	if (
+		! __next40pxDefaultSize &&
+		( otherProps.size === undefined || otherProps.size === 'default' )
+	) {
```

Comment:
There are only four places in this package where we need this logic, so I think we can just copy paste this without abstracting to a utility function.